std/process.d

```diff
@@ -1452,7 +1452,7 @@ void kill(Pid pid, int codeOrSignal)
     else version (Posix)
     {
         import core.sys.posix.signal;
-        if (kill(pid.osHandle, codeOrSignal) == -1)
+        if (core.sys.posix.signal.kill(pid.osHandle, codeOrSignal) == -1)
```

Comment:
Didn't need this for dmd upstream?  I find it hard to believe this import affects no other platform but DragonFly.

Reply:
Reverted... It was needed when running the unittest under dmd-v2.068.1, but apparently not with ltsmaster.